workflow/airflow_libero/script/entrypoint.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+TRY_LOOP="20"
+
+: "${REDIS_HOST:="redis"}"
+: "${REDIS_PORT:="6379"}"
+: "${REDIS_PASSWORD:=""}"
+
+: "${POSTGRES_HOST:="postgres"}"
+: "${POSTGRES_PORT:="5432"}"
+: "${POSTGRES_USER:="airflow"}"
+: "${POSTGRES_PASSWORD:="airflow"}"
+: "${POSTGRES_DB:="airflow"}"
+
+# Defaults and back-compat
+: "${AIRFLOW__CORE__FERNET_KEY:=${FERNET_KEY:=$(python -c "from cryptography.fernet import Fernet; FERNET_KEY = Fernet.generate_key().decode(); print(FERNET_KEY)")}}"
+: "${AIRFLOW__CORE__EXECUTOR:=${EXECUTOR:-Sequential}Executor}"
+
+export \
+  AIRFLOW__CELERY__BROKER_URL \
+  AIRFLOW__CELERY__CELERY_RESULT_BACKEND \
+  AIRFLOW__CORE__EXECUTOR \
+  AIRFLOW__CORE__FERNET_KEY \
+  AIRFLOW__CORE__LOAD_EXAMPLES \
+  AIRFLOW__CORE__SQL_ALCHEMY_CONN \
+
+
+# Load DAGs exemples (default: Yes)
+if [[ -z "$AIRFLOW__CORE__LOAD_EXAMPLES" && "${LOAD_EX:=n}" == n ]]
+then
+  AIRFLOW__CORE__LOAD_EXAMPLES=False
+fi
+
+if [ -n "$REDIS_PASSWORD" ]; then
+    REDIS_PREFIX=:${REDIS_PASSWORD}@
+else
+    REDIS_PREFIX=
+fi
+
+wait_for_port() {
```

Comment:
Might be a useful pattern elsewhere (eg https://github.com/elifesciences/profiles/blob/ff3ca66bf560bbabc430e2eb09a3172ea17affb3/docker-compose.yml#L44).

Reply:
It would be very useful to have this function available everywhere, but short of including it in all base images the alternative solution is to render it from a template (https://elifesciences.atlassian.net/browse/ELPP-3600 or similar)